.github/workflows/static-analysis.yml

```diff
@@ -1,4 +1,4 @@
-name: Static Analysis
+ame: Static Analysis
```

Comment:
```suggestion
name: Static Analysis
```

Reply:
☝️ needs to be fixed before merging so please don't do auto-merge before fixing!